calculator.rb

```diff
@@ -14,6 +14,11 @@ def divide(operand_1, operand_2)
   return operand_1 / operand_2
 end
 
+# Raise operand_1 to the operand_2th power
+def power(operand_1, operand_2)
+  return operand_1 ** operand_2
```

Comment:
Isn't the command `code` only used in classes to switch between different classes? So wouldn't this code take you to a (non-existent) class called "**operand_ ** operand_2**"?